early_access_dunder_init.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from typing import TypedDict
+
+ConnectionType = TypedDict(
+    "ConnectionType",
+    {
+        "hook-class-name": str,
+        "connection-type": str,
+    },
+)
+
+
+ProviderInfoType = TypedDict(
+    "ProviderInfoType",
+    {
+        "package-name": str,
+        "name": str,
+        "description": str,
+        "versions": list[str],
+        "connection-types": list[ConnectionType],
+        "extra-links": list[str],
+    },
+)
+
+
+def get_provider_info() -> ProviderInfoType:
+    return {
+        "package-name": "airflow-provider-datarobot-early-access",
+        "name": "DataRobot Airflow Provider Early Access",
+        "description": "DataRobot Airflow provider early access.",
```

Comment:
These are the only three lines changed versus the regular dunder init

The name and description aren't necessarily needed but it's the "package-name" value that _is_ needed for Apache Airflow discovery mechanisms to work as expected.